.github/workflows/linux.yml

```diff
@@ -38,7 +43,7 @@ jobs:
         uses: actions/checkout@v3
         with:
           repository: godotengine/godot
-          ref: 1a1a450
+          ref: 7e79aea
```

Comment:
Only the SHA doesn't seem to work. can we try `master/7e79aea`?

Reply:
Using the full sha (7e79aead99a53ee7cdf383add9a6a2aea4f15beb) seems to work.